support/flatpak/flatpak_updater.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+set -ex
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if test ! -e "rust-toolchain.toml"; then
+  echo "Please run script in servo git repository"
+  exit
+fi
+
+echo "Syncing Rust dependencies..."
+git clone https://github.com/flatpak/flatpak-builder-tools.git
+{
+  pushd flatpak-builder-tools
+  pushd cargo
+  python3 ./flatpak-cargo-generator.py ../../Cargo.lock \
+    -t -o ../../support/flatpak/cargo-sources.json
+  popd
+
+  pushd pip
+  # hack for missing setuptools_rust
+  {
+    python3 ./flatpak-pip-generator flit_core \
+      -o ../../support/flatpak/python3-flit_core
+    python3 ./flatpak-pip-generator setuptools_rust \
+      -o ../../support/flatpak/python3-setuptools_rust
+  }
+  # other manually listed jsons FIXME remove in future!
+  {
+    # FIXME hardcoded
+    #python3 ./flatpak-pip-generator cryptography \
+    #  -o ../../support/flatpak/python3-cryptography
+
+    python3 ./flatpak-pip-generator pathspec \
+      -o ../../support/flatpak/python3-pathspec
+    python3 ./flatpak-pip-generator pluggy \
+      -o ../../support/flatpak/python3-pluggy
+    python3 ./flatpak-pip-generator tomli \
+      -o ../../support/flatpak/python3-tomli
+    python3 ./flatpak-pip-generator trove_classifiers \
+      -o ../../support/flatpak/python3-trove_classifiers
+    python3 ./flatpak-pip-generator packaging \
+      -o ../../support/flatpak/python3-packaging
+    python3 ./flatpak-pip-generator hatch-vcs \
+      -o ../../support/flatpak/python3-hatch-vcs
+  }
```

Comment:
It would be great to document the mapping all of these dependencies to our Python dependencies. This seems to go well beyond the Python dependencies specified in `python/requirements.txt`. This documentation is necessary so we know which of these to remove when we update our Python requirements, otherwise this will quickly get out of sync and bitrot.